host/src/FunctionsNetHost/Grpc/PathUtils.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Text.Json.Nodes;
+
+namespace FunctionsNetHost.Grpc
+{
+    internal static class PathUtils
+    {
+        /// <summary>
+        /// Gets the absolute path to worker application executable.
+        /// Builds the path by reading the worker.config.json
+        /// </summary>
+        /// <param name="applicationDirectory">The FunctionAppDirectory value from environment reload request.</param>
+        internal static string GetApplicationExePath(string applicationDirectory)
+        {
+            var workerConfigPath = Path.Combine(applicationDirectory, "worker.config.json");
```

Comment:
Let's create an issue to pass the information parsed by the host and avoid reading and parsing this file here.